src/http/routing.rs

```diff
@@ -496,4 +496,16 @@ routes! ('a, {
     StageInstance { channel_id: ChannelId },
     api!("/stage-instances/{}", channel_id),
     Some(RatelimitingKind::Path);
+
+    OAuth2Token,
+    api!("/oauth2/token"),
+    None;
+
+    OAuth2TokenRevocation,
+    api!("/oauth2/token/revoke"),
+    None;
+
+    OAuth2AuthorizationCurrent,
+    api!("/oauth2/@me"),
+    None;
```

Comment:
Could you group these together with the other OAuth route?

Reply:
Sure